### DISCUSSION TITLE
Set up Zustand with remote-redux-devtools from React Native app

### DISCUSSION THREAD

**Discussion by datduyng** [pmndrs/zustand#379] [Q&A]
Hi, 

I am trying to get Zustand to work with a React Native IOS simulator. I realized that middlewares.devtools() won't work out of the box. The problem is that when I launch my RN app through a simulator the `window.__REDUX_DEVTOOLS_EXTENSION__` will be null therefore, the `devtools()` method from the middlewares file won't be able to find the redux extension. I thought that maybe `remote-redux-devtools` can help me solve this problem. Has anyone figured this out?

Thanks,

here is my current code

```jsx
import createStore from 'zustand';
import { devtools, combine } from 'zustand/middleware'

const textStore = (set, get) => ({
    text: 'hellow world',
    otherText: 'other text',
    changeText: (newText) => {
        set(state => ({
            text: newText
        }))
    },
    resetText: (newText) => {
        set({
            text: ''
        })
    }
});

const combineStateCreators = (
    ...stateCreators
) => (set, get, api) => {
    let values = {}

    stateCreators.forEach((sc) => {
        values = Object.assign({}, values, sc(set, get, api))
    })
    return values
}

const store = createStore(devtools(combineStateCreators(
    textStore
)));


```

**Comment by misaku** [comment]
@datduyng  i have same  problem, do you solve this problem?

**Reply by ghost** [reply]
Same problem here :(

**Reply by otaviogaiao** [reply]
same here :/

Any solution?

**Reply by dai-shi** [reply]
Hey folks, thanks for reporting this.

I wish if someone can help on this.
Can anyone dig into it?

**Reply by MindsightsAI** [reply]
Any updates?)
